src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import { User } from '../../types/user';
+
+interface Props {
+  user: User | null;
+}
```

Comment:
You shouldn't render UserInfo component if the User is null. User should be required in this component